### PR TITLE
Update jsonresume-theme-elegant version to latest

### PR DESCRIPTION
**Pull request by maksymgendin**

### CODE REVIEW

File: functions/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "jsonresume-theme-classy": "^1.0.9",
     "jsonresume-theme-cora": "^0.1.1",
     "jsonresume-theme-dave": "^1.0.1",
-    "jsonresume-theme-elegant": "^1.13.2",
+    "jsonresume-theme-elegant": "^1.15.1",
```
Comment (mudassir0909) [inline functions/package.json]: Please update it to `1.16.1` :D 

I've fixed some v1.0 schema breaking changes

**Comment by thomasdavis** [comment]
@mudassir0909 @maksymgendin Does this one still need to be merged?

**Comment by maksymgendin** [comment]
Yes, thanks, @thomasdavis! The deploy to Firebase fails unfortunately 😞

**Comment by thomasdavis** [comment]
Damn't lol not on my box that can deploy it just yet.

**Comment by thomasdavis** [comment]
@maksymgendin  Ahhh, could it's because you didn't `npm i` after adding to `package.json` so it didn't update `package-lock.json`.

Think you could branch off `master`, `npm i` and commit the new package lock file.

Then submit PR and we merge again.

**Comment by maksymgendin** [comment]
@thomasdavis Sorry, I somehow broke `package.json` with this pull request. Don't ask me how this could happen 😅 
I created a new pull request with the fix and the updated `package-lock.json` [here](https://github.com/jsonresume/registry-functions/pull/40)

**Comment by thomasdavis** [comment]
@maksymgendin aha don't worry that much. 

Looks like new build/PR passed. Great job!

**Comment by maksymgendin** [comment]
@thomasdavis One more thing: I see that the JSON schema has changed in version 1.0.0 and due to that fact the example [resume.json](https://github.com/jsonresume/registry-functions/blob/master/functions/resume.json) doesn't work anymore (see missing employer data e.g. [here](https://themes.jsonresume.org/theme/elegant)). Any ideas how we can support multiple versions of the JSON schema? Should we open a new issue for discussions?

**Comment by thomasdavis** [comment]
@maksymgendin Good pick up. The idea with the v1 release was that it will/should break anyone hosted on the registry server, such that theme developers should update accordingly. (not the greatest strat but kinda works) (aka all theme developers are welcome to make breaking changes for v1, and the jsonresume org will take the blame for that) (users who use the suggested hosting service will be alerted to update their resumes to v1) 

Does that answer your concerns or I am missing the question?

**Comment by maksymgendin** [comment]
@thomasdavis Alright then, if this was the idea. Sometimes one have to force people to upgrade in order to decrease technical debt and to push progress 😄 

But still I can imagine that it would be possible to support multiple versions of the schema, depending on the theme's dependency. This may get complicated though.

**Pull request merged**